python/lsst/analysis/ap/apdb.py

```diff
@@ -122,7 +128,7 @@ def _make_flag_exclusion_clause(self, flag_list, column_name='flags'):
         if bitmask == 0:
             warnings.warn(f"Flag bitmask is zero. Supplied flags: {flag_list}", RuntimeWarning)
 
-        return f"(({column_name} & {bitmask}) = 0)"
+        return query.where(table.columns[column_name].op("&")(bitmask) == 0)
```

Comment:
I think the reason why this does not work as expected is because `bitmask` is some `numpy` type which is probably rendered as a bytestring or something (this is a common pitfall with numpy types used as query parameters, I think I had similar issue with `np.int64` instances). Should work better if you convert it to integer first (`int(bitmask)`).

Reply:
Thanks, that fixed it!